README.md

```diff
@@ -22,6 +21,8 @@ Made with ❤👷️ by the team at [.txt](https://dottxt.co)
 
 </div>
 
+Need a high-performance commercial solution for structured outputs? Email us at [contact@dottxt.co](mailto:contact@dottxt.co), or [schedule a call](https://cal.com/team/dottxt/sales).
```

Comment:
I'm wondering if we should have instead "Using structured generation in production or planning to? We'd love your feedback <contact info>"

Reply:
I'd push back on your suggestion, @rlouf -- @cpfiffer's suggestion is better here, it's an invitation directed at a pain point ("high performance").

Reply:
(Though this is internal business, I recognize, feel free to disregard my comment!)